pkg/middlewares/aws_default_subnet.go

```diff
@@ -0,0 +1,53 @@
+package middlewares
+
+import (
+	"github.com/cloudskiff/driftctl/pkg/resource"
+	"github.com/cloudskiff/driftctl/pkg/resource/aws"
+	"github.com/sirupsen/logrus"
+)
+
+// Default Subnet should not be shown as unmanaged as they are present by default
+// This middleware ignores default Subnet from unmanaged resources of they are not managed  IaC
```

Comment:
// This middleware ignores default Subnet from unmanaged resources **if** they are not managed  IaC
:sweat_smile:

Reply:
Would you mind if I fix same typo from VPC middleware in this PR too @moadibfr, it come from a copy paste without enough attention 😅

Reply:
Fixed